packages/twenty-server/src/engine/core-modules/auth/controllers/google-auth.controller.ts

```diff
@@ -33,6 +34,7 @@ export class GoogleAuthController {
 
   @Get('redirect')
   @UseGuards(GoogleProviderEnabledGuard, GoogleOauthGuard)
+  @UseFilters(AuthOAuthExceptionFilter)
```

Comment:
using an adhoc filter to redirect instead of the ApplyCorsToExceptions globalFilter which is not meant to do that